core/gc/src/lib.rs

```diff
@@ -63,7 +65,8 @@ struct GcConfig {
 impl Default for GcConfig {
     fn default() -> Self {
         Self {
-            threshold: 1024,
+            // Start at 1MB, the nursary size for V8 is ~1-8MB and SM can be up to 16MB
+            threshold: 1_000_000,
```

Comment:
Super nitpicky, but since it would be ideal to be 2^n, would it be worthwhile for the threshold to start at `1_048_576`.

Either way, I'm fine with merging.

Reply:
Yes good point